DataGrid: Form Mode unclear description T1103451 (#4172)
DevExpress/devextreme-documentation: concepts/05 UI Components/DataGrid/20 Editing/10 User Interaction/40 Form Mode.md

![DevExtreme HTML5 JavaScript jQuery Angular Knockout UI component DataGrid Editing Form Mode](/images/DataGrid/editing/form_mode.png)

> You can configure the form with the **editing**.[form](/api-reference/10%20UI%20Components/GridBase/1%20Configuration/editing/form.md '/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/editing/#form') object. The DataGrid uses the DevExtreme [Form](/api-reference/10%20UI%20Components/dxForm '/Documentation/ApiReference/UI_Components/dxForm/') UI component. So you can specify [any Form properties](/api-reference/10%20UI%20Components/dxForm/1%20Configuration '/Documentation/ApiReference/UI_Components/dxForm/Configuration/') in the **editing**.[form](/api-reference/10%20UI%20Components/GridBase/1%20Configuration/editing/form.md '/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/editing/#form') object except properties mentioned in description of this object.  

Reviewer: ```suggestion
The DataGrid uses the DevExtreme [Form](/api-reference/10%20UI%20Components/dxForm '/Documentation/ApiReference/UI_Components/dxForm/') UI component. To configure the form, specify [any Form properties](/api-reference/10%20UI%20Components/dxForm/1%20Configuration '/Documentation/ApiReference/UI_Components/dxForm/Configuration/') in the **editing**.[form](/api-reference/10%20UI%20Components/GridBase/1%20Configuration/editing/form.md '/Documentation/ApiReference/UI_Components/dxDataGrid/Configuration/editing/#form') object except properties mentioned in the description of this object. 
```
